src/App.tsx

```diff
@@ -2,108 +2,116 @@ import React from 'react';
 
 import './App.scss';
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+import posts from './api/posts';
```

Comment:
```suggestion
import posts as postsFromServer from './api/posts';
```

Reply:
I just changed the name to postsFromServer, now there is no problem with repeating